src/Juvix/Extra/Paths/Base.hs

```diff
@@ -11,6 +11,9 @@ assetsDirQ = FE.makeRelativeToProject "assets" >>= FE.embedDir
 cssDirQ :: Q Exp
 cssDirQ = FE.makeRelativeToProject "assets/css" >>= FE.embedDir
 
+juvixStyQ :: Q Exp
+juvixStyQ = FE.makeRelativeToProject "latex/juvix.sty" >>= FE.embedFileRelative
```

Comment:
Do you need to use both `FE.makeRelativeToProject` and `FE.embedFileRelative` ? `FE.embedFileRelative` will work with the relative path?

Reply:
done